lib/alerts_viewer_web/live/alerts_to_close_live.html.heex

```diff
@@ -21,57 +21,89 @@
   />
 </div>
 
-<.table id="bus_alerts" sticky={true} rows={Keyword.keys(@alerts_by_route)}>
-  <:col :let={route} label="Route Name">
-    <%= Routes.get_by_id(@bus_routes, Atom.to_string(route)) |> Route.name() %>
+<.table id="bus_alerts" sticky={true} rows={@sorted_alerts}>
+  <:col :let={alert} label="Alert Id (sorted by duration)">
+    <%= alert.id %>
   </:col>
-  <:col :let={route} label="Alert Ids (sorted by duration)">
-    <%= @alerts_by_route[route] |> Enum.map(& &1.id) |> Enum.join(", ") %>
+  <:col :let={alert} label="Route Names">
+    <.informed_entity_icons entities={route_names_from_alert(alert, @bus_routes)} />
```

Comment:
In the `route` field in the informed entities on alerts, is that the route name or the id? I think I was assuming it's the ID but I don't know.  I also don't know if the people viewing this are expecting the ID or the name.

Reply:
It's the ID.

I don't know for sure about the user expectations, but my guess would be that displaying the name on the page is friendlier but that they are also knowledgeable enough to understand the IDs. If you look at the [list of bus routes](https://www.mbta.com/schedules/bus), for the vast majority of them, the ID and the name are the same thing. The only ones where it differs are the 6 Silver Line and 2 Crosstown routes, so it's a pretty limited distinction.

Reply:
I think we should use the `short_name` value rather than the `id`, which we tend to do when showing it in badges on the website, in Skate, and Alerts UI.

Sometimes we'll preface "SL1" with "Silver Line - ", but I don't think we ought to do that here.

![image](https://github.com/mbta/alerts_viewer/assets/772939/040bfabc-1637-4372-be9b-f7d5cc88b5de)
![image](https://github.com/mbta/alerts_viewer/assets/772939/441efcd0-7206-456d-ac19-39da56bb7857)

Reply:
I might keep this the way it is, then, since it'll show the name if it's available. (For Steve, it'll show the short name if available, then the long name if that's not there, then the id if neither are there.) 

I gotta say, score 1 for the new tests, that's the only reason I caught that this was showing IDs.